### PR TITLE
Moved scaffolding contents to its approppriate place

### PR DESCRIPTION
**Pull request by jmoreira-valory**
## Proposed changes

Moved scaffolding contents to its approppriate place. Rewritten some parts of the text.

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/guides/create_service_from_scratch.md [hunk 1]
```diff
@@ -1,2 +1,87 @@
-!!! info
-    This tutorial will be added soon.
+Developing an agent service with the {{open_autonomy}} framework sometimes requires to build a custom agent for it, especially when the business logic of existing agents do not fit for the functionality of the service. This involves reusing or creating the packages that the new agents need: skills, connections, protocols, etc. This can seem a bit overwhelming at the beginning, as new developers might not be familiar with the structure of each one of those.
+
+To simplify the development, the framework provides a scaffold tool that allow developers speed up their flow by auto-generating functional, skeleton classes with all the boilerplate code in place. Therefore, you can focus on implementing the actual business logic of the service.
```
Comment (DavidMinarsch) [inline docs/guides/create_service_from_scratch.md:3]:
```suggestion
To simplify the development, the framework provides a scaffold tool that allow developers speed up their flow by auto-generating functional, skeleton classes with all the boilerplate code in place. Therefore, you can focus on implementing the actual business logic of the skill, that is part of the service.
```

Comment (DavidMinarsch) [reply docs/guides/create_service_from_scratch.md:3]: or something along those lines

File: docs/guides/create_service_from_scratch.md [hunk 2]
```diff
@@ -1,2 +1,87 @@
...
+    You will see that the generated rounds, payloads and behaviours already appear with their correct names, as well as the `HelloWorldAbciApp` and its transition function.
+
+4. **Fill in the business logic code of the service.** By default, the generated rounds, payloads and behaviours are initialized to empty values. It is your turn to define what actions are occurring at each state of the service. You can review how a number of [demo services](../demos/index.md) are implemented, or read more about the internals of [{{fsm_app}}s](../fsm_app_introduction.md)).
+
+5. **Push the generated skill to the local registry.** Now that you have successfully created the skill, you can push it to the local registry.
+
+    To use the local registry, the framework expects that a `packages` directory exists at the same level as our agent. In this example, it expects that the directory `my_workspace/packages` exists. If you don't have a local registry, you can create it as
+    ```bash
+    mkdir ../packages
+    ```
```
Comment (DavidMinarsch) [inline docs/guides/create_service_from_scratch.md:68]: I think the guide of setting up local registry already exists elsewhere?

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]
I think there's an opportunity to simplify a bit more

File: docs/guides/create_service_from_scratch.md (hunk 1 above)
Comment (jmoreira-valory) [reply docs/guides/create_service_from_scratch.md:3]: fixed

**Comment by jmoreira-valory** [comment]
The 'working with packages' guide will be polished in an upcoming PR.

**Comment by jmoreira-valory** [comment]
The 'working with packages' guide will be polished in an upcoming PR.

**Review by DavidMinarsch — APPROVED** [review]

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]
CI failing atm - o/w lgtm

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**